File: Python/1164.py
for i in range(int(input())):
    ent = int(input())
    soma = 0

    for x in range(1, ent):
        if ent % x == 0:
            soma += x

    if ent == soma:
        print(f'{ent} eh perfeito')
    else:
        print(f'{ent} nao eh perfeito')

File: C/1164.c
#include <stdio.h>
 
int main() {
 
    int n;
    scanf("%d", &n);
    for(int i = 0; i < n; i++){
        int num, soma = 0;
        scanf("%d", &num);
        for(int y = 1; y < num; y++){
            if(num%y == 0){
                soma += y;
            }
        }
        if(num == soma){
            printf("%d eh perfeito\n", num);
        }else{
            printf("%d nao eh perfeito\n", num);
        }
    }
}
